bugcatcher/ftl.py

```diff
@@ -31,11 +31,10 @@
                       re.compile('^__'),
                       re.compile('/venv/'),
                       re.compile('^venv/'),
-                      re.compile('^\.'),
```

Comment:
These 2 lines were giving me problems. Now that we are parsing `.gitignore` files, I think we can allow these `\.` files. Their `.gitignore` can ignore anything they wouldn't want shared. We could also append all `.gitignore` pattern lists with `.*`.